### DISCUSSION TITLE
Cannot read properties of null (reading 'content')

### DISCUSSION THREAD

**Discussion by nongtiny** [vercel/next.js#70135] [Help]
### Summary

Hi,

I found this error on the production of my website. From what I checked, the error came from this file line 119 

https://github.com/vercel/next.js/blob/canary/packages/next/src/client/head-manager.ts#L119

Has anyone got this error same as me or know an approach to solve this?

Thanks in advanced.

**Comment by Enkratia** [comment]
@nongtiny probably `headCountEl` is equal to `null`. Yout can't read properties of `null` like this: `null.content`. `null` has no property `content`.

**Comment by chonlatit-tpit** [comment]
@Enkratia @nongtiny 
Should we create Pull Request to fixed  this  ?

from : 
```
const headCount = Number(headCountEl.content)
```
to: 
```
const headCountElStr = headCountEl?.content;
const headCount = headCountElStr ? Number(headCountElStr) : 0;
 ```
